schedule/migration/migration_offline_media_x86_64.yaml

```diff
@@ -1,6 +1,6 @@
 name: migration_offline_x86_64.yaml
```

Comment:
Suggest to create a new schedule file for libyui test, ex: schedule/migration/migration_offline_media_x86_64_libyui.yaml

Reply:
Why? If the old schedule is used nowhere, why keep it around?

Reply:
I'd reserve the non-libyui test for comparation, anyway, Jonathon, WDYT?

Reply:
The point of this MR is to get rid of needle-based modules because they require more maintenance, if we keep both, we need even more maintenance, so I'm rather against, also that would require an additional MR. restapi is already a proven concept.

Reply:
Sounds good.